index.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Umuhanzi concert home page</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+     rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet">
+    <script src="https://kit.fontawesome.com/3ffbe7c408.js" crossorigin="anonymous"></script>
+</head>
+<body>
+<header>
+    <i class="material-icons menu-btn">menu</i>
+</header>
+<nav class="navbar navbar-expand-sm bg-body navbar-light  shadow p-2 mb-5 rounded align-content-start  ">
+     
+    <div class="container-fluid  justify-content-start px-5">
+        <div class="container w-50 d-none d-lg-flex">
+    <a href="index.html" class=" navbar-brand logo btn"><i class="material-icons md-orange">audiotrack</i>Umuhanzi</a>
+  </div>
+    <i class="material-icons close-btn">close</i>
+      <ul class="navbar-nav">
+        <li class="nav-item p-2">
+          <a class="nav-link text-dark" href="index.html">Home</a>
+        </li>
+        <li class="nav-item p-2">
+          <a class="nav-link text-dark" href="about.html">About</a>
+        </li>
+        <li class="nav-item p-2">
+          <a class="nav-link text-dark" href="#artists">Artists</a>
+        </li>
+        <li class="nav-item p-2">
+            <a class="nav-link btn btn-menu" target="_blank" href="#"><i class="fas fa-play-circle px-2"></i>Stream Live</a>
+          </li>
+        <li class="nav-item p-2">
+          <a class="nav-link btn btn-menu2" target="_blank" href="#">Buy/Book Ticket</a>
+        </li>
+      </ul>
+    </div>
+  
+  </nav>
```

Comment:
- [ ] Good job working on this project. Your page header is looking good but it misaligns a bit from the guided design.
It will be good if you could please implement the dark header on top the white header

Your Design |
:---------------:|
![e1](https://user-images.githubusercontent.com/39523312/148261366-05909126-994a-40fa-b77b-9239cd80fbae.png) 

Template Design |
:----------------------:|
![e2](https://user-images.githubusercontent.com/39523312/148261625-72aa45e8-3dfc-4e61-82c8-4f6899a61db4.png)